plugins/pathable.cpp

```diff
@@ -293,7 +378,10 @@ static bool wagon_flood(color_ostream &out, unordered_set<df::coord> * wagon_pat
         TRACE(log,out).print("checking tile: (%d, %d, %d); pathability group: %d\n", pos.x, pos.y, pos.z,
             Maps::getWalkableGroup(pos));
 
-        if (entry_tiles.contains(pos)) {
+        // Ensure our wagon flood end points lands on an edge tile.
+        // When there is no path to the depot entry_tiles will not
+        // contain any edge tiles.
```

Comment:
this statement is not quite correct. when there is no path to the depot, then the *search_edge* will never contain any edge tiles. `entry_tiles` contains edge tiles regardless of the accessibility of the depot.